### ISSUE TITLE
Description for 'Typography'

### ISSUE THREAD

**Issue by bzupancic** [six7/figma-tokens#86]
It would be great if "description":  "" were available for Typography, like it is for colors. I try to document when/where/why for each text style and color style within Figma. So being able to add it via the figma-tokens plugin would be awesome!

**Comment by six7** [comment]
Just noticed we introduced a bug there with a recent update.

Description currently is supported in typography styles, the UI however exposes it in a faulty way that the plugin then can't handle. If your JSON looks like this it will work at the moment.

```
{
  "typography": {
    "H1": {
        "fontFamily": "Inter",
        "fontWeight": "Regular",
        "lineHeight": "AUTO",
        "fontSize": "24",
        "description": "Some style description"
    }
  }
}
```

However, when generating a style from scratch, it will output values like this, which is OK, but the plugin should be able to parse that when translating that to styles:

```
"H2": {
      "value": {
        "fontFamily": "Inter",
        "fontWeight": "Regular",
        "lineHeight": "AUTO",
        "fontSize": "24"
      },
      "description": "Does not work"
    }
 ```

**Comment by six7** [comment]
Fixed this with a couple other bugs in #88
